analyzer/src/main/kotlin/managers/utils/MavenSupport.kt

```diff
@@ -262,6 +262,7 @@ class MavenSupport(workspaceReader: WorkspaceReader) {
 
         populator.populateFromSettings(request, settings)
         populator.populateDefaults(request)
+        repositorySystemSession.injectProxy(request)
```

Comment:
Commit message: Please remove the dot from the end of the commit message title, and move the `Resolves #2905.` part to just before the `Signed-off-by` footer.